src/envo/envo-edit.owl

```diff
@@ -39106,6 +39194,75 @@ AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03510041>
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03520003> (cloud part)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03520003> "A part of a cloud."@en)
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000116> <http://purl.obolibrary.org/obo/ENVO_03520003> "This definition is practical but may need refinement."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_03520003> <http://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03520003> "cloud part"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03520003> <http://purl.obolibrary.org/obo/ENVO_01001784>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03520003> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/ENVO_01000760>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03520004> (cloud base)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03520004> "A cloud part which is closest to the surface of the astronomical body local to the cloud."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_03520004> <http://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03520004> "cloud base"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03520004> <http://purl.obolibrary.org/obo/ENVO_03520003>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03520005> (cloud top)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03520005> "A cloud part which is furthest from the surface of the astronomical body local to the cloud."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_03520005> <http://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03520005> "cloud top"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03520005> <http://purl.obolibrary.org/obo/ENVO_03520003>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03520006> (supercooled water droplet)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://www.skybrary.aero/index.php/Supercooled_Water_Droplets>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03520006> "A water droplet which exists in liquid form at temperatures below 0°C (32°F)."@en)
```

Comment:
I'd trim down the def to only mention the temperature differentia, as the liquid state is inherited

Reply:
I've trimmed it and added the temperature in Kelvin instead of °F.
> A water droplet which exists at temperatures below 0°C (273.15K)